### ISSUE TITLE
Usar Serviços de Entrega padrão quando não estiver configurado no produto

### ISSUE THREAD

**Issue by rebecapf** [pedro-teixeira/correios#175]
Habilitei a opção de Filtro de Serviços por Produto e agora tenho que configurar os serviços de entrega por produto... até aí ok, pois tenho produtos de 300g que não podem ser enviados por carta registrada e tenho produtos que podem. 
O problema é... eu cadastro os produtos utilizando um ERP que é integrado com o magento e tem informações que eu tenho que atualizar depois, direto no magento, pois no ERP não tem essas informações, como por exemplo os serviços de entrega. Então nesse meio tempo, eu gostaria que o produto assumisse as configurações padrões que defini nas formas de entrega, assim é possível o cliente calcular o frete do produto sem problemas, assim como é feito com os tamanhos da embalagem, que quando estão em branco no produto, assumem os tamanhos padrões definidos nas formas de entrega.

É possível fazer isso? 

Desde já, agradeço!

**Comment by rafaelpatro** [comment]
O _serviço por produto_, se ativado, precisa ser configurado no produto. Infelizmente não há configuração padrão.
O que vc pode fazer é configurar todos os produtos em uma única operação em massa. Mas isso é um recurso do Magento.

**Comment by rafaelpatro** [comment]
Se fizer da seguinte forma, acho q vai resolver seu problema:

1. Nas configurações da extensão, seleciona somente os serviços que vai precisar.
2. No catálogo de produtos, clica em Atualizar Atributos.
3. No atributo _Serviços de Entrega_, marca todas as opções, e salva.

Dessa forma, somente os serviços que vc selecionou nas configurações da extensão serão exibidos.
Como a configuração da extensão é obrigatória, vc só pode desmarcar os serviços que não quer em cada produto.

**Comment by rebecapf** [comment]
Lá nos atributos não tem nenhuma opção cadastrada, ele deve pegar alguma configuração... será q tenho que por manualmente?

**Comment by rafaelpatro** [comment]
Sim. Configurar o atributo do produto é uma operação manual. Lembrando que,
para os Serviços, as configurações da extensão são obrigatórias. Mesmo que
vc selecione todos os serviços, em todos os produtos, somente os serviços
da extensão serão exibidos.

Em 02/08/2016 14:39, "rebecapf" <notifications@github.com> escreveu:
